search-app/src/cmr/search/services/community_usage_metrics/metrics_service.clj

```diff
@@ -167,14 +171,31 @@
    params
    [(partial cpv/validate-boolean-param :comprehensive)]))
 
+(defn- aggregate-usage-metrics
+  "Combine access-counts for entries with the same short-name."
+  [metrics]
+  (let [java-map (new java.util.HashMap)]
```

Comment:
I think this could be done a lot cleaner with `reduce`

Reply:
I reverted the function back to its previous version with minor modifications.  Can't tell you how it works, but it's definitely clojurish and provides the right outputs!

Reply:
We are not supposed to check in any code that we can't read. :) The above code is more efficient than `group-by`, but who cares.